Donation-platform/src/main/java/com/google/FindTrendingCharities.java

```diff
@@ -38,16 +46,24 @@
   final double TAGS_SCORE_WEIGHT = 0.75;
   final double AVG_REVIEW_WEIGHT = 0.25;
 
+
   // returns the collection of top trending charities
   public Collection<Charity> query() {
-    DbSetUpUtils setUp = new DbSetUpUtils();
-    db = setUp.getDbCalls();
-    // setUp.dbSetUp();                                      //only call once
-    // TODO: Change to getAllCharities() when integrating db
+    ds = DatastoreServiceFactory.getDatastoreService();
+    db = new DbCalls(ds);
+    DbSetUpUtils setUp = new DbSetUpUtils(ds, db);
```

Comment:
move this to constructor so that its only called once?

Reply:
done.